modules/restore-replace-crashlooping-etcd-member.adoc

```diff
@@ -144,6 +144,15 @@ sh-4.2# etcdctl member list -w table
 +
 You can now exit the node shell.
 
+. Turn off the quorum guard by entering the following command:
++
+[source,terminal]
+----
+$ oc patch etcd/cluster --type=merge -p '{"spec": {"unsupportedConfigOverrides": {"useUnsupportedUnsafeNonHANonProductionUnstableEtcd": true}}}'
+----
++
+That command ensures that you can successfully re-create secrets and roll out the static pods.
```

Comment:
Also I think I've seen the term `roll out` before in kubernetes docs, but I think its used at one word, like `rollout`. Wdyt?

Reply:
As I understand it, `rollout` as one word is a noun and `roll out` as two words is a verb. I think in this instance, the two-word spelling is correct.

Reply:
Gotcha, That makes sense! Is this kinda the same context in the [kub docs](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) just for curiosity 😄